resources-generator/src/main/kotlin/dev/icerock/gradle/generator/android/AndroidImagesGenerator.kt

```diff
@@ -38,22 +44,47 @@ class AndroidImagesGenerator(
             files.map { key to it }
         }.forEach { (key, file) ->
             val scale = file.nameWithoutExtension.substringAfter("@").substringBefore("x")
-            val drawableDirName = "drawable-" + when (scale) {
-                "0.75" -> "ldpi"
-                "1" -> "mdpi"
-                "1.5" -> "hdpi"
-                "2" -> "xhdpi"
-                "3" -> "xxhdpi"
-                "4" -> "xxxhdpi"
+            val drawableDirName = "drawable" + when (scale) {
+                "0.75" -> "-ldpi"
+                "1" -> "-mdpi"
+                "1.5" -> "-hdpi"
+                "2" -> "-xhdpi"
+                "3" -> "-xxhdpi"
+                "4" -> "-xxxhdpi"
                 else -> {
-                    println("ignore $file - unknown scale ($scale)")
-                    return@forEach
+                    if (file.svg) {
+                        ""
+                    } else {
+                        logger.info("ignore $file - unknown scale ($scale)")
+                        return@forEach
+                    }
                 }
             }
 
             val drawableDir = File(resourcesGenerationDir, drawableDirName)
             val processedKey = processKey(key)
-            file.copyTo(File(drawableDir, "$processedKey.${file.extension}"))
+
+            val resourceExtension = if (file.svg) "xml" else file.extension
+            val resourceFile = File(drawableDir, "$processedKey.$resourceExtension")
+            if (file.svg) {
+                parseSvgToVectorDrawable(file, resourceFile)
+            } else {
+                file.copyTo(resourceFile)
+            }
+        }
+    }
+
+    private fun parseSvgToVectorDrawable(svgFile: File, vectorDrawableFile: File) {
+        try {
+            vectorDrawableFile.parentFile.mkdirs()
+            vectorDrawableFile.createNewFile()
+            FileOutputStream(vectorDrawableFile, false).use { os ->
+                Svg2Vector.parseSvgToXml(svgFile, os)
+                    .takeIf { it.isNotEmpty() }
+                    ?.let { error -> logger.warn("parse from $svgFile to xml:\n$error") }
```

Comment:
but if we can't parse - it should be error too, not warn. why you think it warn?

Reply:
`parseSvgToXml` returns a warning or an error message. Sometimes we get a message and the vector drawable is still created. So I picked `warn`, but if you think in this case `error` is a better option. We can go with that.

Reply:
ah, ok. result of `Svg2Vector.parseSvgToXml` is error String, so your logic is correct. `warn` level is correct